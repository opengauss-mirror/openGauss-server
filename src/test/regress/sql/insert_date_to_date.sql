create table tt(col1 date);
insert into tt values('10000-01-28');
insert into tt values(TO_DATE('10000-01-28','yyyy-mm-dd'));
drop table tt;
