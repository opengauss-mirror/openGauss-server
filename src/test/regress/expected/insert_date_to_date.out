create table tt(col1 date);
insert into tt values('10000-01-28');
ERROR:  invalid data for "year =  10000", value must be between -4712 and 9999, and not be 0
LINE 1: insert into tt values('10000-01-28');
                              ^
CONTEXT:  referenced column: col1
insert into tt values(TO_DATE('10000-01-28','yyyy-mm-dd'));
ERROR:  invalid data for "year =  10000" ,value must be between 1 and 9999, and not be 0
CONTEXT:  referenced column: col1
drop table tt;
